src/bootstrap/src/core/sanity.rs

```diff
@@ -195,8 +193,22 @@ than building it.
         if !["A", "B", "C"].contains(&target_str.as_str()) {
             let mut has_target = false;
 
-            let supported_target_list =
-                output(Command::new(&build.config.initial_rustc).args(["--print", "target-list"]));
+            let supported_target_list: HashSet<String> =
+                output(Command::new(&build.config.initial_rustc).args(["--print", "target-list"]))
```

Comment:
Noting that we are doing this inside of a for loop, while we only need to execute it once. Invoking rustc multiple times could have non-trivial impact on bootstrap startup time if multiple targets are passed.

Reply:
Right